tests/test_metrics.py

```diff
@@ -894,21 +894,30 @@ def test_sum_distances(self):
                       [1, 0, 0]])
 
         # Dataframe with multiple columns
-        metric_1 = SumDistancesConnectedNeurons(positions)
```

Comment:
it will be nice to have some tests for the indirected case as well.

Reply:
in the last one (metric 5) is_directed=False :) I can do more if needed